Sources/Sentry/SentrySDK.m

```diff
@@ -202,6 +203,7 @@ + (void)startWithOptions:(SentryOptions *)options
 {
     startOption = options;
     [SentryLog configure:options.debug diagnosticLevel:options.diagnosticLevel];
+    sentry_initializeAsyncLogFile();
```

Comment:
`m`: Why did you move this here from the SentryLog?

Reply:
If the new visionOS configuration, we can't call C functions from Swift anymore.
And since we need this for visionOS as well, we have to have a new place to start asyncLog.

Reply:
I guess SentryLog depends on this? Isn't this now super easy to break? I don't know that these two belong together. I wold prefer moving this `sentry_initializeAsyncLogFile` to an ObjC wrapper and then calling it from Swift.

Reply:
> sentry_initializeAsyncLogFile to an ObjC wrapper

We can do this, but SentryLog does not depends on it. We probably call it from SentryLogs as a central place where we start both log mechanisms.

Reply:
OK, even though `SentryLog` doesn't depend on it, it's now easy to get it wrong when changing it the next time. Previously, it wasn't. A comment would also be sufficient I guess.